build.sh

```diff
@@ -50,6 +51,25 @@ check_managed_prereqs()
     fi
 }
 
+install_dotnet_cli()
+{
+    echo "Installing the dotnet/cli..."
+    hash wget 2>/dev/null || { echo >&2 "Please ensure curl is present before running this script"; exit 1; }
```

Comment:
https://github.com/dotnet/cli/blob/5f5e3ad74c0c1de7071ba1309dca2ea289691163/scripts/install.sh

This is a heavy hammer for us to use. We ourselves depend on wget in our repo. See in the same file.